lib/msc_extensions/extension_audio_events/audio_events.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:matrix/matrix.dart';
+
+extension AudioEventExtension on Event {
+  /// returns if the event is a voice message
+  bool get isVoiceMessage =>
+      content['org.matrix.msc2516.voice'] != null ||
+      content['org.matrix.msc3245.voice'] != null;
```

Comment:
Can we maybe instead check if this is a `Map<String,Object?>`? If someone puts in an `int` in this or something else, I think I would expect that this getter returns false. What do you think?

Reply:
sure, sounds good to me.